db/views/latest_blood_pressures_per_patient_per_months_v01.sql

```diff
@@ -0,0 +1,15 @@
+SELECT DISTINCT ON (patient_id, year, month)
```

Comment:
This `DISTINCT` assures that a patient only shows up once in each month, right? If we didn't use `DISTINCT`, I believe a patient would show up multiple times in a month if they had multiple BP recordings in the month. Now that we're using `DISTINCT`, which reading wins? Do we have control over that? Does the most recent reading win because of the ordering clause?

Reply:
Yes, the ordering + distinct ensures we get the last BP for each month